mllib/src/main/scala/org/apache/spark/mllib/tree/RandomForest.scala

```diff
@@ -443,84 +272,4 @@ object RandomForest extends Serializable with Logging {
   @Since("1.2.0")
   val supportedFeatureSubsetStrategies: Array[String] =
     Array("auto", "all", "sqrt", "log2", "onethird")
```

Comment:
These are now hard coded in two places. I think it would be better to have this call `RandomForestParams.supportedFeatureSubsetStrategies` in ml.